src/assets/javascripts/helpers/youtube/youtube.js

```diff
@@ -91,14 +91,13 @@ let redirects = {
       "https://piped.kavin.rocks",
       "https://piped.silkky.cloud",
       "https://piped.tokhmi.xyz",
+      "https://piped.moomoo.me",
       "https://il.ax",
       "https://piped.syncpundit.com",
       "https://piped.mha.fi",
```

Comment:
@ManeraKai I didn't remove it is ordered as in the Official list. See this line.